src/seedsigner/views/seed_views.py

```diff
@@ -159,11 +159,12 @@ def run(self):
     Loading seeds, passphrases, etc
 ****************************************************************************"""
 class LoadSeedView(View):
-    SEED_QR = ButtonOption("Scan a SeedQR", SeedSignerIconConstants.QRCODE)
+    SEED_QR = ButtonOption(" Scan a SeedQR", SeedSignerIconConstants.QRCODE)
```

Comment:
The space char is fine to sorta fix the presentation issues caused by the icons in the buttons being different widths.

But I'd drop a TODO here that the real solution is to update `Button` and/or `ButtonListScreen` to do a proper text alignment.